### PR TITLE
Custom refresh control for Android

### LINKED ISSUE #114: refreshing, onRefresh are not working
"react": "16.9.0"
"react-native": "0.61.4"
"react-native-draggable-flatlist": "https://github.com/computerjazz/react-native-draggable-flatlist.git"
Jan 8 2020

### PR DESCRIPTION
**Pull request by Peretz30**
Resolves #114 #174 #135 
Added custom implementation for pull to refresh for android. Check example in `README`. Works fine for my project, but test it please for yours apps.

### CODE REVIEW

**Comment by nikkundra** [comment]
Has there been any update on this PR from April 28?

**Comment by AbdelkhalekESI** [comment]
Still not working on android

File: README.md [hunk 1]
```diff
@@ -107,3 +111,96 @@ class Example extends Component {
...
+    return (
+      <View style={{ flex: 1 }}>
+        <DraggableFlatList
+          data={this.state.data}
+          renderItem={this.renderItem}
+          keyExtractor={(item, index) => `draggable-item-${item.key}`}
+          onDragEnd={({ data }) => this.setState({ data })}
+          refreshControl={Platform.OS === 'ios' ? iOSrefreshControl : androidRefreshControl}
+          onRefresh={this.onRefresh}
+          refreshing={this.state.refreshing}
```
Comment (lucasfeijo) [inline README.md:197]: Why not access these two props from the same RefreshControl object you provide for iOS?

Comment (Peretz30) [reply README.md:197]: > Why not access these two props from the same RefreshControl object you provide for iOS?

dude, it was 1 year ago. I dont remember anything about this now.

**Pull request closed without merging**